Zend/zend_observer.c

```diff
@@ -72,12 +72,18 @@ ZEND_API void zend_observer_fcall_register(zend_observer_fcall_init init) {
 ZEND_API void zend_observer_startup(void) {
 	zend_llist_init(&zend_observers_fcall_list, sizeof(zend_observer_fcall_init), NULL, 1);
 	zend_llist_init(&zend_observer_error_callbacks, sizeof(zend_observer_error_cb), NULL, 1);
+
+	zend_observer_fcall_op_array_extension = -1;
```

Comment:
This fixes the bug

Reply:
Hmm, I'm not sure how setting this to `-1` on startup is different than setting it statically in the globals. Any insight there?

Reply:
minit stage may be re-entered in forked children